.autover/changes/0f890f57-dfe4-4e16-b537-dddc9a0c2be0.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Projects": [
+    {
+      "Name": "AWS.AspNetCore.DistributedCacheProvider",
+      "Type": "Patch",
+      "ChangelogMessages": [
+        "Update AWS SDK to Preview 11"
```

Comment:
Change log needs to call out the removal of .NET 6

Reply:
should it still be a patch then or change to minor?

Reply:
I'm not sure how it matters since I think we end up reworking the v4 branch and main so that we GA off of the v4 codebase and that will be 1.0 when it GAs. But for now you could set it to `minor` to signify the change was significant.